ui/src/lib.rs

```diff
@@ -47,7 +47,7 @@ pub enum TUIExitReason {
 mod op_effects;
 use op_effects::stack_indices_affected;
 
-pub struct Tui {
+pub struct Tui<'a> {
```

Comment:
is there a scenario where we need the source code afterward?

if not let's not introduce lifetimes